### ISSUE TITLE
Java Debugger Crashing after a few Seconds

### ISSUE THREAD

**Issue by LukeJelly** [Microsoft/vscode-java-debug#549]
Running a very simple java program and debugging the debugger seems to work fine for about 30 seconds then I get the following errors:
"[error] Error: Language client is not ready yet "
"[error] Cannot read property 'path' of undefined: TypeError: Cannot read property 'path' of undefined"
"[error] Running the contributed command:'java.execute.workspaceCommand' failed.: Error: Running the contributed command:'java.execute.workspaceCommand' failed."

The debugger crashes I no longer have the option to step through and my variable values in the top left of my screen disappear. Soon after that I will get the error "The language Support for Java server crashed 5 times in the last 3 minutes" I've uploaded a .zip file of the logs.

I have tried launching as Administrator, Uninstalling and Reinstalling both VS Code and my JDK, restarting the computer, uninstalling and reinstalling all extensions, disabling my anti-virus, disconnecting from the internet and installing different versions of my JDK.

##### Environment
- Operating System: Win 10 PRO 1809(OS BUILD 17763.316)
- JDK version: Java Version 11.0.1 2018-10-16 LTS
- Visual Studio Code version: Visual Studio Code 1.31.1(user setup)
- Java extension version: Unknown(Redhat is 0.40.0)
- Java Debugger extension version: 0.17.0

##### Steps To Reproduce
1. Open java program
2. Set breakpoints on line(92 of Test.java)
3. Run program, wait 30 seconds

This is the Assignment I was working on when I first noticed it but it happens in any project now.
This is a simple Polymorphic/Inheritance Project for a CSC142 Class.
[HW06.zip](https://github.com/Microsoft/vscode-java-debug/files/2944026/HW06.zip)

##### Additional Informations
I attempted to recreate this issue on my Surface Book Laptop
OS Version Win 10 PRO 1803(OS BUILD 17134.590)
Visual Studio Code 1.31.1(user setup)
Java Version 11.0.1 2018-10-16 LTS
Same versions for extensions.

The debugger seemed to work fine on my laptop and I have no other computers to try and recreate this on.

I'm pretty new to VS Code but from the research I've done it looks like this has been an issue with other people but those always seem to lead to a dead end. Either there fix doesn't work for me or they are no longer responding so the issue gets closed.

I'm new here so sorry if this is in the wrong spot. Does anyone have any ideas on how to fix this?

Thanks for your help in advance

My logs.
[20190306T191446.zip](https://github.com/Microsoft/vscode-java-debug/files/2944031/20190306T191446.zip)
[20190306T193828.zip](https://github.com/Microsoft/vscode-java-debug/files/2944032/20190306T193828.zip)

**Comment by snjeza** [comment]
@LukeJelly You may want to take a look at https://github.com/redhat-developer/vscode-java/issues/828#issuecomment-470182218 and https://github.com/redhat-developer/vscode-java/issues/815#issuecomment-469826697

**Comment by LukeJelly** [comment]
@snjeza  thanks for the quick reply.  I just ran a sfc /scannow and it came back with nothing.  I also double checked that my PATH environmental variable was set correctly and it all looked normal.  I then tried deleting the entire %UserProfile%.vscode\extensions\redhat.java- file and restarting.  As of right now I have a feeling that mite of fixed it.  The debugger has been sitting for over 30seconds and hasn't crashed.  I'm going to let it sit for an hour and if it doesn't crash i'll come back and let you know!
